PULL_REQUEST_TEMPLATE

```diff
@@ -5,20 +5,33 @@ Please include a summary of the change and which issue is fixed. Please also inc
 List any dependencies that are required for this change.
 -->
 
-## Pull Request Checklist
+## Submitter Checklist:
 
-- [ ] My patch has gone through review and I have addressed review comments
-- [ ] My patch has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!`
+- [ ] Submitted a [ticket](https://github.com/brave/brave-ios/issues) for my issue if one did not already exist.
+- [ ] My patch or PR title has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!` (or `No Bug: <message>` if no relevant ticket)
+- [ ] *Unit Tests* are updated to cover new or changed functionality
+- [ ] User-facing strings use `NSLocalizableString()`
+- [ ] New files have MPL-2.0 license header.
 
-- [ ] I have updated the *Unit Tests* to cover new or changed functionality
-- [ ] I have updated the *UI Tests* to cover new or changed functionality
-- [ ] I have marked the bug with `[needsuplift]`
-- [ ] I have made sure that localizable strings use `NSLocalizableString()`
 
-## Screenshots
+## Test Plan:
 
-If your patch includes user interface changes that you would like to suggest or that you would like UX to look at, please include them here.
+<!-- Any useful notes for reviewer explaining how best to test and verify. -->
 
-## Notes for testing this patch
+### Screenshots:
+
+<!-- If your patch includes user interface changes that you would like to suggest or that you would like UX to look at, please include them here. -->
+
+
+## Reviewer Checklist:
+
+- [ ] PR is linked an issue via [Zenhub](https://www.zenhub.com/extension).
+- [ ] Issues are assigned to at least one epic.
+- [ ] Issues include necessary QA labels:
+  - [ ] `QA/(Yes|No)`
+  - [ ] `release-notes/(include|exclude)`
+  - [ ] `bug` / `enhancement`
+- [ ] Necessary security reviews have taken place.
```

Comment:
Not sure that it matters, but perhaps we can add a note about the crawled labels (`security`, `sec-low`, `sec-high`)

Reply:
Don't think that would matter at this point too much, because once a ticket is closed, the crawler is probably irrelevant at that point. 🤷‍♂️